package.json

```diff
@@ -145,10 +146,11 @@
     "prettier": "^1.14.3",
     "pretty-json-stringify": "^0.0.2",
     "puppeteer": "^1.19.0",
-    "terser": "^4.2.0",
+    "terser": "^5.3.8",
     "typescript": "3.9.7"
   },
   "dependencies": {
+    "acorn-node": "^1.7.0",
```

Comment:
is this supposed to be a devDep?

Reply:
ya! I was just about to do it earlier but then VSCode froze and I forgot. Has vscode been freezing for you lately? It's a new thing for me...last week, few times a day I get a 100% cpu usage

Reply:
hm not really, are you using insiders too?